thoth/common/openshift.py

```diff
@@ -217,7 +217,7 @@ def parse_python_solver_name(cls, solver_name: str) -> Dict[str, Any]:
                 f"Python version encoded into Python solver name does not start with 'py' prefix: {solver_name!r}"
             )
 
-        python_version = ".".join(list(python_version))
+        python_version = python_version[0] + "." + python_version[1:]
```

Comment:
Can you use f-string instead of concatenation?

Reply:
Adjusted, PTAL 👍🏻